src/linear_algebra/dimension.lean

```diff
@@ -147,6 +147,26 @@ lemma dim_span_set {s : set V} (hs : linear_independent K (λ x, x : s → V)) :
   dim K ↥(span K s) = cardinal.mk s :=
 by rw [← @set_of_mem_eq _ s, ← subtype.val_range]; exact dim_span hs
 
+lemma cardinal_le_dim_of_linear_independent
+  {ι : Type v} {v : ι → V} (hv : linear_independent K v) :
```

Comment:
Again, two different `v`s on the same line...

Reply:
I fixed it throughout the whole file.